pyproject.toml

```
@@ -0,0 +1,49 @@
+[build-system]
+requires = ["setuptools>=77", "setuptools_scm>=7", "wheel"]
```

changing this so tests work on the python3.8 build
```suggestion
requires = ["setuptools>=75", "setuptools_scm>=7", "wheel"]
```